src/test/java/io/cdap/delta/bigquery/BigQueryUtilsTest.java

```diff
@@ -350,4 +365,45 @@ private static Set<SourceTable> generateSourceTableSet(int noOfTables) {
     }
     return allTables;
   }
+
+  private static Page<Table> generateBQTablesPage(int num) {
+    Page<Table> pg = new Page<Table>() {
+      @Override
+      public boolean hasNextPage() {
+        return false;
+      }
+
+      @Override
+      public String getNextPageToken() {
+        return null;
+      }
+
+      @Override
+      public Page<Table> getNextPage() {
+        return null;
+      }
+
+      @Override
+      public Iterable<Table> iterateAll() {
+        List<Table> tableList = new ArrayList<>();
+
+        for (int i = 1; i <= num; i++) {
+          // Create Table
+          Table tableMock2 = Mockito.mock(Table.class);
+
+          String tableName = TABLE_PREFIX + i;
+          TableId tableId = TableId.of("testproject", DATASET, tableName);
```

Comment:
instead of using hardcoded "testproject" , 
Declare a `private static final String PROJECT = "testproject";` just like `DATASET` and use it .

Reply:
Okay